rose-admin

```diff
@@ -39,11 +40,11 @@ class Client(object):
         try:
             method = getattr(self, method_name)
         except AttributeError:
-            self.usage('Unknown verb: %r' % verb)
+            self.usage(f'Unknown verb: {verb}')
         try:
             method()
         except Exception as e:
-            log.error("Error running: %s, %s", verb, e)
+            log.error(f"Error running: {verb}, {e}")
```

Comment:
This is bad change. Logging message should not be formatted. Formatting the messages means we always pay for formatting the logs even when the logs are not enabled (e.g. log.debug() when log level is WARNING).

log.error() is likely always loged, but it does not make sense to handle differently different log messages, so logging calls always pass the arguments to the logger and the logger format the messages before they are written.

Please remove all the changes in logging calls.